test_acl.py should be skipped for PTF32 topology

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/Azure/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**
test_acl.py fails when run in PTF32 topology although the testcase says 'any' for topology support.
pytestmark = [
    pytest.mark.acl,
    pytest.mark.disable_loganalyzer,  # Disable automatic loganalyzer, since we use it for the test
    pytest.mark.topology("any"),
]
<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Steps to reproduce the issue:**
1. Run test_acl.py on ptf32 topology


**Describe the results you received:**
Test case fails because the ACL table that the testcase expects to create is never created.
Log analyzer expects 'Created ACL table' string to be present in syslog but it is not found because ACL table creation fails.
           # Cleanup Config DB if table creation failed
            logger.error("ACL table creation failed, attempting to clean-up...")
            create_or_remove_acl_table(duthost, acl_table_config, setup, "remove")
>           raise err
E           LogAnalyzerError: {'match_messages': {'/tmp/syslog.2022-04-28-08:29:12': []}, 'total': {'expected_match': 0, 'expected_missing_match': 1, 'match': 0}, 'match_files': {'/tmp/syslog.2022-04-28-08:29:12': {'expected_match': 0, 'match': 0}}, 'expect_messages': {'/tmp/syslog.2022-04-28-08:29:12': []}, 'unused_expected_regexp': ['.*Created ACL table.*']}

acl/test_acl.py:321: LogAnalyzerError
------------------------------------------------------------------------------------------------ Captured log setup ------------------------------------------------------------------------------------------------
INFO     root:__init__.py:49 Completeness level not set during test execution. Setting to default level: CompletenessLevel.basic
INFO     root:__init__.py:139 Test has no defined levels. Continue without test completeness checks
================================================================================================= warnings summary =================================================================================================
/usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:538
  /usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:538: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: tests.common.dualtor
    self.import_plugin(import_spec)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
===================================================================================== 1 warnings, 736 error in 101.66 seconds ======================================================================================
Exception AttributeError: "'NoneType' object has no attribute 'close'" in <bound method EventDescriptor.__del__ of <ptf.ptfutils.EventDescriptor instance at 0x7fcd1ca47128>> ignored
**Describe the results you expected:**
Test case is expected to pass (or be skipped, not error or fail)

**Additional information you deem important:**
ACL Table creation fails because it is created with 'empty' port list and results in CLI failure.
From the testscript, the ACL table is created in the following way:
"config acl add table {} {} -s {} -p {}".format(
                    acl_table_config["table_name"],
                    acl_table_config["table_type"],
                    acl_table_config["table_stage"],
                    ",".join(setup["acl_table_ports"][namespace]),

And since 'acl_table_ports' is empty, there are no arguments after '-p' option to create the ACL table and hence the CLI command fails to create ACL table.

From reading the setup fixture, it is clear that acl_table_ports is populated only in the case of T0 or T1 topologies. In case of PTF32, it is populated using upstream_ports which is again filled in only for T0 or T1 topologies. For PTF32 topologies, there are no upstream ports.

Please confirm if this testcase is expected to be run on PTF32. And if not, please pytest mark the topology in the testcase as such.
<!--
software/ASIC/Hardware Flatform version and info
-->

--- a/tests/acl/test_acl.py
+++ b/tests/acl/test_acl.py
@@ -27,7 +27,7 @@ logger = logging.getLogger(__name__)
 pytestmark = [
     pytest.mark.acl,
     pytest.mark.disable_loganalyzer,  # Disable automatic loganalyzer, since we use it for the test
-    pytest.mark.topology("any"),
+    pytest.mark.topology("t0", "t1", "t2", "m0", "mx"),
 ]
 
 BASE_DIR = os.path.dirname(os.path.realpath(__file__))

